client/client.js

```diff
@@ -10,6 +10,7 @@ function getRequest(e) {
     let search = e.target.searchBar.value;
     let res = fetch(`https://www.googleapis.com/customsearch/v1?key=AIzaSyBARGZeFhbu-VGkHidrPQVe8UxPwHSXIY0&cx=007675054066069573056:uzm9atbkkps&q=${search}&num=10`)
         .then(r => r.json())
+        // .then(hideNavBar)
```

Comment:
for line 23 I've been told that innerHTML can lave you vulnerable to attacks? Maybe consider finding a different implementation?